recipe/meta.yaml

```diff
@@ -16,7 +16,7 @@ source:
 
 build:
   skip: true  # [win]
-  number: {{build}}
```

Comment:
My complaint is that it is not honoring the build variable we had in the template! Having the build variable close to the version variable is really good, otherwise we may forget to fix bump the build in a future manual rerendering.

Reply:
I don't think it's hard to remember to bump the build number. It's still in the first 20 lines of the file.

#10 removes the unused variable